pymc_experimental/model_builder.py

```diff
@@ -258,6 +254,23 @@ def build_model(
         """
         raise NotImplementedError
 
+    def save_model_coords(self, X: Union[pd.DataFrame, pd.Series], y: pd.Series):
```

Comment:
Why does save_model_coords require both X and y as inputs?

Reply:
I did that because I thought there may be some highly unusual cases in which y is also required. I would have no problem with it myself if this function only takes X.